### PR TITLE
ci: Move all jobs to private runners

### PR DESCRIPTION
**Pull request by lluiscampos**

### CODE REVIEW

**Review by oldgiova — APPROVED** [review]

**Review by oldgiova — APPROVED** [review]

**Review by kacf — APPROVED** [review]

**Comment by lluiscampos** [comment]
The jobs are running into other sorts of errors. I'll revert back to just updating the public shared runners tag and wait for a final decision on moving or not to private runners.

**Comment by lluiscampos** [comment]
> (...) I'll revert back to just updating the public shared runners tag (...)

See https://github.com/mendersoftware/mender-artifact/pull/611

**Pull request closed without merging**
